.github/CODEOWNERS

```diff
@@ -1,10 +0,0 @@
-* @cyberark/community-and-integrations-team @conjurinc/community-and-integrations-team @conjurdemos/community-and-integrations-team
-
-# Changes to .trivyignore require Security Architect approval
-.trivyignore @cyberark/security-architects @conjurinc/security-architects @conjurdemos/security-architects
-
-# Changes to .codeclimate.yml require Quality Architect approval
-.codeclimate.yml @cyberark/quality-architects @conjurinc/quality-architects @conjurdemos/quality-architects
-
-# Changes to SECURITY.md require Security Architect approval
-SECURITY.md @cyberark/security-architects @conjurinc/security-architects @conjurdemos/security-architects
```

Comment:
It's unclear from the docs whether the CODEOWNERS file is pulled and used.  Given each project is likely to have a different set between C&I and Core, I think preserving the CODEOWNERS file is best here